### PR TITLE
add alt text for images

### LINKED ISSUE #70: Can't set an alt on the header image
Go to: http://output.jsbin.com/muhuru.
The header image does not have any alt text.

### PR DESCRIPTION
**Pull request by greenyouse**
Fixes #70

### CODE REVIEW

File: paper-card.html [hunk 1]
```diff
@@ -164,6 +165,14 @@
         },
 
         /**
+         * The text alternative of the card's title image.
+         */
+        alt: {
+          type: String,
+          value: null
```
Comment (notwaldorf) [inline paper-card.html]: `value` is by default null, so this isn't needed

**Comment by notwaldorf** [comment]
LGTM, one small comment left

**Comment by greenyouse** [comment]
OK, it should be all set

**Comment by notwaldorf** [comment]
Thanks so much!

**Pull request merged**